### ISSUE TITLE
Language/grammar issues in Fitness Trend User Guide

### ISSUE THREAD

**Issue by conorsleith** [thomaschampagne/elevate#714]
### Step 1: Are you in the right place?

  * Your problem may have been already covered in the FAQ. Did you check https://github.com/thomaschampagne/elevate/wiki/Frequently-Asked-Questions?
  * Your problem or feature request may already exists in project issues section. Did you check https://github.com/thomaschampagne/elevate/issues?

### Step 2: Describe your environment

  * Plugin version: 6.80 Stable
  * Chrome/Opera/Chromium version: Chrome 71.0.3578.98
  * OS version: MacOS 10.14
  
### Step 3: Describe the problem:
The wording of the Fitness Trend User Guide could be improved
#### Steps to reproduce:

  1. Click Fitness Trend
  2. Click (?) button
  3. Read
  
#### Actual Behavior:

n/a
  
#### Expected Behavior:

n/a
  
#### Media attachement:
Propose the following wording:

In simple terms, the fitness trend is a kind of "black box system" that models the response of your body exposed to various efforts performed over time. Your body reacts to these stimuli with "fatigue" and creates a "background fitness base" that makes you "stronger" for the subsequent efforts. To fully use the acquired fitness, you have to be in "form," meaning that you have overcome fatigue with rest and recovery.

To put numbers on fitness, fatigue and form over time, each effort has to be quantified. This is where "stress scores" are used.

There exist several recognized stress score calculation methods depending on available sensors. These stress scores are detailed in a further section.

**Comment by thomaschampagne** [comment]
@conorsleith Thanks for your help :) English is my second language :)

It will be in next release

**Comment by conorsleith** [comment]
You're welcome! Elevate is wonderful :)
